extensions/geode-modules/src/main/java/org/apache/geode/modules/session/catalina/ClientServerSessionCache.java

```diff
@@ -139,17 +137,7 @@ public boolean isClientServer() {
 
   @Override
   public int size() {
-    // Add a single dummy key to force the function to go to one server
-    Set<String> filters = new HashSet<String>();
-    filters.add("test-key");
-
-    // Execute the function on the session region
-    Execution execution = FunctionService.onRegion(getSessionRegion()).withFilter(filters);
-    ResultCollector collector = execution.execute(RegionSizeFunction.ID);
-    List<Integer> result = (List<Integer>) collector.getResult();
-
-    // Return the first (and only) element
-    return result.get(0);
+    return getSessionRegion().sizeOnServer();
```

Comment:
Does this mean we can delete the RegionSizeFunction in a future release?

Reply:
Yes, we can. I actually created [GEODE-6924](https://issues.apache.org/jira/browse/GEODE-6924) yesterday for removing `RegionSizeFunction` once this PR gets merged.